src/dmd/cparse.d

```diff
@@ -4134,6 +4136,7 @@ final class CParser(AST) : Parser!AST
         SCW scw;        /// storage-class specifiers
         MOD mod;        /// type qualifiers
         AST.Expressions*  alignExps;  /// alignment
+        structalign_t packalign = STRUCTALIGN_DEFAULT;  // #pragma pack alignment value
```

Comment:
Perhaps /// comments for this one as well.

Reply:
ok